libs/components/datetime/src/lib/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -45,6 +45,10 @@ const SKY_DATEPICKER_VALIDATOR = {
   multi: true,
 };
 
+/**
+ * Input directive that provides date input functionality and handles date parsing, formatting and
```

Comment:
```suggestion
 * Provides date input functionality for the datepicker and handles date parsing, formatting, and
```
In the other descriptions that I found in our docs for components and directives, we use sentence fragments that start with a verb, so it seems like we should delete "Input directive that" for consistency with that pattern.

I'm not sure what prompted adding the missing description here, but should we also add a similar description for SkyFuzzyDatepickerInputDirective? And in the existing description for SkyDatepickerComponent, should

"You must wrap this component around an input with the skyDatepickerInput directive."

be

"You must wrap this component around an input with the skyDatepickerInput or SkyFuzzyDatepickerInputDirective directive."